src/layers/affineScalingLayer.jl

```diff
@@ -10,8 +10,11 @@ kron(e3,kron(b2,e1)) + kron(e3,kron(e2,b1));
 mutable struct AffineScalingLayer{T} <: AbstractMeganetElement{T}
     nData       # describe size of data, at least first two dim must be correct.
```

Comment:
Attribute should be typed. I think in this case it will need to be a `Vector{Int}`?